### PR TITLE
Updated the module to use the new moreinfo web-service

### PR DESCRIPTION
**Pull request by cableman**
The module have been updated with support for front-covers only. A more advanced version have to be developed later that takes back-covers etc.

### CODE REVIEW

File: lib/addi-client/AdditionalInformationService.php [hunk 1]
```diff
@@ -52,66 +52,66 @@ protected function sendRequest($identifiers)
...
-  	
+
     $authInfo = array('authenticationUser' => $this->username,
                       'authenticationGroup' => $this->group,
                       'authenticationPassword' => $this->password);
-    $client = new SoapClient($this->wsdlUrl);
-    
-    $startTime = explode(' ', microtime()); 
-    $response = $client->additionalInformation(array(
+    $client = new SoapClient($this->wsdlUrl . '/moreinfo.wsdl');
```
Comment (kasperg) [inline lib/addi-client/AdditionalInformationService.php:59]: I do not think it is wise to assume where the WSDL is located and what is is called (`$this->wsdlUrl . '/moreinfo.wsdl'`) although I'm aware of the fact that it is the case in Ding 1.8.

Instead I think we should prompt the user for the WSDL url.

**Comment by kasperg** [comment]
If revisting this code I think we should rename classes and variable so that they no longer refer to AdditionalInformation/addi and instead the new MoreInfo service.

Referencing something obsolete is just confusing.

**Comment by cableman** [comment]
I agree but I think that should be in another PR. Mostly because we don't have the time now and DBC is closing the old service.

I think a lighthouse ticket about refactoring this module is need, so it can be part of part II.

File: lib/addi-client/AdditionalInformationService.php (hunk 1 above)
Comment (cableman) [inline lib/addi-client/AdditionalInformationService.php:59]: I'll make the changes

**Comment by cableman** [comment]
I will close this PR and make a new one based on the suggestions

**Pull request closed without merging**